src/main/java/monitoringsystemturbo/presenter/timeline/TimelineView.java

```diff
@@ -107,4 +118,49 @@ public void showDay(Integer day) {
         currentDay = day;
     }
 
+    void addTimeLine(Timeline timeLine) {
+
```

Comment:
decide if you add new lines after method name or not. Almost everywhere there is no newline so I suggest not adding it, but the more import thing is to be consistent